app/javascript/components/patient/PatientPage.js

```diff
@@ -28,7 +30,7 @@ class PatientPage extends React.Component {
             Monitoree Details {this.props.patient.user_defined_id ? `(ID: ${this.props.patient.user_defined_id})` : ''}{' '}
             {this.props.patient.id && <a href={window.BASE_PATH + '/patients/' + this.props.patient.id + '/edit'}>(edit details)</a>}
             <span className="float-right collapse-hover">
-              <i className="fas fa-bars"></i>
+              {this.state.hideBody ? <FontAwesomeIcon icon={faChevronDown} /> : <FontAwesomeIcon icon={faChevronUp} />}
```

Comment:
is there a reason why we can't use the existing font awesome library instead?

Reply:
yes - the font-awesome javascript library doesn’t rerender on a React rerender trigger (I'm using state change here) but the react font awesome library does.  I talked to @kylietmo about this and ideally we'd like to swap all our react components to use this anyway